sw/device/sca/lib/sca.c

```diff
@@ -293,3 +293,17 @@ void sca_call_and_sleep(sca_callee callee, uint32_t sleep_cycles) {
   OT_DISCARD(dif_clkmgr_gateable_clock_set_enabled(
       &clkmgr, CLKMGR_CLK_ENABLES_CLK_IO_DIV4_PERI_EN_BIT, kDifToggleEnabled));
 }
+
+/**
+ * PRNG for for masking key.
```

Comment:
```suggestion
 * PRNG for masking key.
```

Reply:
Oops, thanks for noticing. It's fixed now.